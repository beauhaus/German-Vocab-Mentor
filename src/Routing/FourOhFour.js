import React from 'react';
import {Link} from 'react-router-dom';


const FourOhFour = () => (
    <div>
        <h1>OOOPS! 404</h1>
        <h2><Link to="/">GO HOME?</Link></h2>
    </div>
)

module.exports = FourOhFour;