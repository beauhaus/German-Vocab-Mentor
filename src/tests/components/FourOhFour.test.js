import React from 'react';
import {shallow} from 'enzyme';
import FourOhFour from '../../Routing/FourOhFour'


test('should render dashboard correctly',() => {
    const wrapper = shallow(<FourOhFour/>)
    expect(wrapper).toMatchSnapshot();
})