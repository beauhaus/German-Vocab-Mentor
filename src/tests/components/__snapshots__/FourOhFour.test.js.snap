// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly 1`] = `
<div>
  <h1>
    OOOPS! 404
  </h1>
  <h2>
    <Link
      replace={false}
      to="/"
    >
      GO HOME?
    </Link>
  </h2>
</div>
`;
